### ISSUE TITLE
Improve typography for readability

### ISSUE THREAD

**Issue by theecrit** [openoakland/openoakland.org#115]
## Description
The current site typography uses out-of-the-box css that doesn't adhere to OpenOakland brand standards and makes content hard to scan and read easily. We'd like to update the type site-wide to reflect the latest recommendations in the [OpenOakland Style Guide](https://docs.google.com/document/d/1SvhwrQcu4mNMS05zE4AIXA8zsc7qpUeWVA9GvgiZaHo/edit#heading=h.pq1z4e7y1jfo):

- [ ] H1 header in Open Sans Extra Bold: {color: #000; font-size:1.5rem; line-height:1; margin-top:0;}
- [ ] H2 header in Open Sans: {color: #013743; font-size:1.25rem; line-height:1; margin-top:1.9rem}
- [ ] H3 header in Open Sans: {color: #013743; font-size:1rem; line-height:1.1; margin-top:1.9rem}
- [ ] H4 header in Open Sans: {color: #013743; font-size:1rem; line-height:1.1; margin-top:1.9rem}
- [ ] H5 header in Open Sans: {color: #xx; font-size:1rem; line-height:1.2; margin-top:1.9rem}
- [ ] paragraph text: {color: #000; font-size:1rem; line-height:1.3; margin-top:20px}
- [ ] unordered list style should be set with {margin-top:1.6rem;}
- [ ] li styles should be set with {margin-top:1.3rem;} 
- [ ] class="caption": {color: #013743; font-size:.7rem; line-height:1; margin-top:1.3rem;}

### Notes
- We're using `rem` here to ensure sizing is consistently proportionate to the root font-size on the site, rather than the active element. The goal is to help reinforce consistency.
- There are ongoing discussions in #oo-website around choosing a distinct display face for headers but it would be useful to update the styles with the above recommendations at least temporarily for the sake of site readability. We can always update again once design decisions are finalized.

#### If you are working on this issue, consider the following ongoing issues:
- [ ] "Purify" Jekyll source files. Make the file you're working on all markdown, or all HTML, and not both.
